addons/portal/static/src/scss/portal.scss

```diff
@@ -486,7 +486,7 @@ form label {
 
     .o_portal_page_size {
         width: 100%;
-        max-width: 210mm; // A4 page's width
+        max-width: 235mm;
```

Comment:
Why this change?

Reply:
Spacing issue,mention in pad  - https://drive.google.com/file/d/1ZNXotoO9wjZ5iE9SF_z6Hj0ScNBQRD-4/view?usp=drivesdk